doc/source/parameters/cfd/post_processing.rst

```diff
@@ -20,12 +20,18 @@ This subsection controls the post-processing other than the forces and torque on
     set calculate average velocities  = false
     set initial time                  = 0.0
 
-    # Pressure drop calculation
+    # Pressure and momentum drop calculations
     set calculate pressure drop     = false
     set pressure drop name          = pressure_drop
+    set calculate momentum drop     = false
+    set momentum drop name          = momentum_drop
     set inlet boundary id           = 0
     set outlet boundary id          = 1
```

Comment:
Why not merge the momentum drop into the pressure drop file directly? I feel it's not necessarily a good idea to create more and more files and more and more options.  You could just add the momentum drop directly when you calculate pressure drop to the same file and that would be it. Would make it less parameters, less logic and less files and anyway when. you calculate the pressure drop you will most likely want to calculate the momentum drop.